samza-rest/src/main/java/org/apache/samza/rest/proxy/job/YarnCliJobStatusProvider.java

```diff
@@ -1,154 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.apache.samza.rest.proxy.job;
-
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import org.apache.hadoop.yarn.api.records.YarnApplicationState;
-import org.apache.samza.SamzaException;
-import org.apache.samza.rest.model.Job;
-import org.apache.samza.rest.model.JobStatus;
-import org.apache.samza.rest.script.ScriptOutputHandler;
-import org.apache.samza.rest.script.ScriptPathProvider;
-import org.apache.samza.rest.script.ScriptRunner;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-
-/**
- * An implementation of the {@link JobStatusProvider} that retrieves
- * the job status from the YARN command line interface.
- */
-public class YarnCliJobStatusProvider implements JobStatusProvider {
```

Comment:
Check with @jmakes if it's OK to just delete this class. Do we need to keep it around for compatibility?

Reply:
@prateekm, we chatted about it. Looks like this new impl is better in every way. Ok to delete!